LEGO1/mxdsfile.h

```diff
@@ -1,20 +1,46 @@
 #ifndef MXDSFILE_H
 #define MXDSFILE_H
 
-class MxDSFile
+#include "mxcore.h"
+#include "mxstring.h"
+#include "mxioinfo.h"
+#include "mxdssource.h"
+class MxDSFile : public MxDSSource
 {
 public:
-  __declspec(dllexport) MxDSFile(const char *,unsigned long);
-  __declspec(dllexport) virtual ~MxDSFile();
-  __declspec(dllexport) virtual long Close();
-  __declspec(dllexport) virtual unsigned long GetBufferSize();
-  __declspec(dllexport) virtual unsigned long GetStreamBuffersNum();
-  __declspec(dllexport) virtual long Open(unsigned long);
-  __declspec(dllexport) virtual long Read(unsigned char *,unsigned long);
-  __declspec(dllexport) virtual long Seek(long,int);
+  __declspec(dllexport) MxDSFile(const char *filename, unsigned long skipReadingChunks);
+  __declspec(dllexport) ~MxDSFile();
+  __declspec(dllexport) long Open(unsigned long);
+  __declspec(dllexport) long Close();
+  __declspec(dllexport) long Read(unsigned char *,unsigned long);
+  __declspec(dllexport) long Seek(long,int);
+  __declspec(dllexport) unsigned long GetBufferSize();
+  __declspec(dllexport) unsigned long GetStreamBuffersNum();
```

Comment:
Why did you remove the `virtual` from these? We know from the DLL exports that these are definitely `virtual`.

Reply:
I guess I could leave the virtual on here too but they're now virtual by default thanks to the base class having them as virtual methods.